win32ss/user/ntuser/caret.c

```diff
@@ -439,11 +439,11 @@ NtUserShowCaret(HWND hWnd OPTIONAL)
       RETURN(FALSE);
    }
 
-   if (Window) UserRefObjectCo(Window, &Ref);
```

Comment:
Looking at that function name, I realized.... Aren't those "thread locks" basically what our original "RefObjectCo" meant to be?